index.html

```diff
@@ -1,16 +1,72 @@
 <!DOCTYPE html>
 <html>
+    
+    <input type="text" id="mySearch" onkeyup="myFunction()" placeholder="Search.." title="Type in a category">
+
+<ul id="myMenu">
+  <li><a href="file:///Users/osvaldoescobar/Projects/Thinkment/philosophical_issues.html">Philosophical Issues</a></li>
+  <li><a href="file:///Users/osvaldoescobar/Projects/Thinkment/clashes_between_ideas.html">Clashes between Ideas</a></li>
+
+
+    
     <head>
+        <style> 
+        
+#mySearch {
+  width: 20%;
+  font-size: 14px;
+  padding: 9px;
+  border: 4px solid #ddd;
+}
+
+#myMenu {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+#myMenu li a {
+  padding: 10px;
+  text-decoration: none;
+  color: black;
+  display: block
+}
+
+#myMenu li a:hover {
+  background-color: #eee;
+}
```

Comment:
Move [all the lines within the style tags](https://github.com/Osvaldo178/Thinkment/pull/6/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R14-R37) into the style.css folder you have in the main folder: https://github.com/Osvaldo178/Thinkment/blob/main/style.css